packages/client/src/OsdkObjectFrom.ts

```diff
@@ -30,16 +28,19 @@ export type OsdkObjectPrimaryKeyType<
 
 type OsdkCommonFrom<
   Q extends ObjectTypeDefinition<any> | InterfaceDefinition<any, any>,
-  L extends ObjectOrInterfacePropertyKeysFrom2<Q> =
-    ObjectOrInterfacePropertyKeysFrom2<Q>,
+  P extends keyof Q["properties"] | "all" = "$all",
```

Comment:
Are the `all`s supposed to match here?

Reply:
They are, looks like its only fixed in the follow on. Good eye